doc/guides/security-steward-on-off-boarding.md

```diff
@@ -0,0 +1,22 @@
+# Security Steward Onboarding/OffBoarding
+
+## Onboarding
+
+* Confirm the new steward agrees to keep all private information confidential
+  to the project and not to use/disclose to their employer.
```

Comment:
* Add them to the security-stewards team in the GitHub nodejs-private
  organization.